docs/about/whats_new.rst

```diff
@@ -40,7 +37,7 @@ Massive cleaning effort across the codebase, spearheaded by @pjonsson, with a fo
 Other Changes
```

Comment:
I think line 21 is easier to read if "It has" is replaced with "GridWorkflow has".